sdk-base/src/main/java/com/mapbox/common/MapboxCommonLogger.kt

```diff
@@ -0,0 +1,14 @@
+package com.mapbox.common
+
+internal object MapboxCommonLogger {
+
+  internal fun internalLogE(tag: String, message: String) = Log.error(message, "$SDK_IDENTIFIER\\$tag")
```

Comment:
Nit: could remove internal from internal object

Reply:
then in another file it will look a bit strange imho
```
fun logE(tag: String, message: String) = logE(tag, message)
```
let's leave it to showcase more clearly that logger we are using from common is not exposed (it's package-private)